src/core.c/Int.pm6

```diff
@@ -26,6 +26,10 @@ my class Int does Real { # declared in BOOTSTRAP
         )
     }
 
+    multi method ACCEPTS(Int:D: Int:D \other, --> Bool:D) {
+        self == other
```

Comment:
This is a similar check to `ACCEPTS(Str:D: Str:D)`, which comes alongside the [coercive `ACCEPTS(Str:D: Any:D)`](https://github.com/rakudo/rakudo/blob/master/src/core.c/Str.pm6#L111-L113). `Num` is also lacking in its own `ACCEPTS`.